.appcast.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <rss xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle" xmlns:dc="http://purl.org/dc/elements/1.1/" version="2.0">
   <channel>
+    <item>
```

Comment:
`skpm` will do this for you in one step – it'll bump the version, update the manifest, create a github release, and publish – so i'd avoid doing it manually here.

Reply:
Ahh this was done by skpm. Was trying to release a pre-release version to give it a spin. Will remove the commits from this branch and the release.

Reply:
cool. once the actual functionality is merged you can do a `skpm publish 1.0.1 --skip-registry`